internal/rest/resources/cluster.go

```diff
@@ -554,6 +546,14 @@ func clusterMemberDelete(s *state.State, r *http.Request) response.Response {
 		return response.SmartError(err)
 	}
 
+	// Remove the cluster member from the database.
```

Comment:
Is there a reason `ResetClusterMember()` is called twice? Once before removing the cluster member from the DB and a second time after removing it from the trust store down below.

Would this mean the `PreRemove` hook is called a second time?

Reply:
It's the same as `AddClusterMember`, based on the cluster notification request header it either runs the hook or the default implementation.

Reply:
Ah I see, thanks!